src/setup/index.js

```diff
@@ -42,6 +43,12 @@ export default function () {
     I18nManager.forceRTL(false);
     I18nManager.swapLeftAndRightInRTL(false);
 
+    // Disable font scaling for react native Text/TextInput components because we are handling it manually
+    RNText.defaultProps = RNText.defaultProps || {};
+    RNText.defaultProps.allowFontScaling = false;
+    RNTextInput.defaultProps = RNTextInput.defaultProps || {};
+    RNTextInput.defaultProps.allowFontScaling = false;
```

Comment:
Doesn't quite resolve the original concern I had. These should be moved our custom Text and TextInput components vs. adding these default here.

https://github.com/Expensify/App/blob/main/src/components/Text.js
https://github.com/Expensify/App/blob/main/src/components/TextInput/BaseTextInput.js#L18

Reply:
Ok